### PR TITLE
Ml 105 participation system database

### PR DESCRIPTION
**Pull request by GXZheng312**
# Description

Please include a summary of the change, which issue is fixed and the context if the new feature. List any dependencies that are required for this change.

## Type of change

Please delete options that are not relevant and add new options if necessary. 

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings

### CODE REVIEW

File: backend/repository/paticipation.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
+const mongoose = require('mongoose');
```
Comment (KevinVanH) [inline backend/repository/paticipation.js:1]: Waarom alle calls in één file en niet gescheiden zoals bij de rest van de APIs?

**Review by KevinVanH — CHANGES_REQUESTED** [review]

**Review by KevinVanH — CHANGES_REQUESTED** [review]
Opzich goed, maar ik vraag me af waarom je gekozen hebt alle API calls in één file te doen ipv zoals bij de andere in losse files

File: backend/repository/paticipation.js (hunk 1 above)
Comment (Veaxx) [inline backend/repository/paticipation.js:1]: verander bestandsnaam paticipation -> participation

**Review by Veaxx — CHANGES_REQUESTED** [review]

File: backend/repository/paticipation.js (hunk 1 above)
Comment (GXZheng312) [reply backend/repository/paticipation.js:1]: Omdat het al best wel overzichtelijk is om in een bestand te doen

**Pull request merged**